src/main/java/com/parasoft/findings/utils/doc/remote/RulesRestClient.java

```diff
@@ -128,17 +145,41 @@ public RuleInfo(String sId, String sAnalyzerId, String sAnalyzerVersion, String
         }
     }
 
-    private static final String DOC_PATH = "doc"; //$NON-NLS-1$
+    public static class CreationResult {
```

Comment:
Is there any benefits we can get by creating this class instead of adding a "status" variable for the client?